src/Components/Server/src/Circuits/DefaultCircuitFactory.cs

```diff
@@ -54,13 +54,6 @@ internal class DefaultCircuitFactory : CircuitFactory
             jsRuntime.Initialize(client);
             componentContext.Initialize(client);
 
-            var authenticationStateProvider = scope.ServiceProvider.GetService<AuthenticationStateProvider>() as IHostEnvironmentAuthenticationStateProvider;
-            if (authenticationStateProvider != null)
-            {
-                var authenticationState = new AuthenticationState(httpContext.User); // TODO: Get this from the hub connection context instead
-                authenticationStateProvider.SetAuthenticationState(Task.FromResult(authenticationState));
-            }
-
```

Comment:
This whole block was redundant - we were initializing the authentication state provider twice.

Also the logic in this copy was the old and debunked variant (relying on `HttpContext`).